benchmarks/python/test_broadcast_add_fwd.py

```diff
@@ -112,9 +112,14 @@ def test_bcast_add_baseline_benchmark(
         x = x.t()
     assert x.is_contiguous() == contiguous
 
+    benchmark_fn = {
+        "eager": bcast_add_fwd_fn,
+        "torchcompile": torch.compile(bcast_add_fwd_fn),
+    }
```

Comment:
Personally I'd avoid using pure strings as much as possible because it'd be difficult to debug when things don't work as intended. For example, I suppose you won't get any error even if you typed "torch.compile". 

Not a request to change anything, but just my two cents.

Reply:
we 've seen a similar typo err where `device` was typed as `degvice`

Reply:
Agreed, I considered using enums, but the misspelling can cause errors with that too.
Here, if `torchcompile` is spelled as `torch.compile`,  in later lines, `benchmark_fn[executor]` will error out. So combined with the assertion that @liqiangxl suggested in the below comment for `get_test_executor`, we should be able to catch any spelling errors.

Please let me know if you have any suggestions which may be more error-proof.